### ISSUE TITLE
Breaking change should have been a major version update

### ISSUE THREAD

**Issue by GeoffreyEmerson** [davipon/esbuild-plugin-pino#78]
Firstly, thank you for your work! 🏆

tl;dr - Version 1.3.0 should have been a major bump to 2.0.0.

What I recently learned: Semver has this annoying aspect where the "magic zero" packages can have breaking changes in the "minor" updates. This is accounted for in the use of the carat, where packagers won't download v0.17.0 if you specify "^0.16.0". But using the carat on a package that has a major version greater than 0 will allow packagers to download increased minor versions.

The issue: `esbuild-plugin-pino` does not use the Zero major version, so those of us using the carat specifier will get v1.3.0 downloaded when we refresh our packages, which does not correspond to esbuild versions below 0.17.0.

Even though esbuild-plugin-pino v1.3.0 is only adapting to a minor version update in another package, that adaptation was a breaking change, and so the version bump should have been a major version update to 2.0.0.

Suggestion: Consider deleting 1.3.0 from the registry and resubmitting it as 2.0.0.
